alethzero/MainFace.cpp

```diff
@@ -54,6 +74,18 @@ void Plugin::addAction(QAction* _a)
 	m_main->addAction(_a);
 }
 
+QAction* Plugin::addMenuItem(QString _n, QString _menuName, bool _sep)
+{
+	QAction* a = new QAction(_n, main());
+	QMenu* m = main()->findChild<QMenu*>(_menuName);
+	if (_sep)
+		m->addSeparator();
+	m->addAction(a);
+	return a;
+}
+
+std::vector<std::function<Plugin*(MainFace*)>>* MainFace::s_linkedPlugins = nullptr;
```

Comment:
standard states that all static literal initialisations take place before static non-literal initialisations.